package.json

```diff
@@ -8,7 +8,15 @@
     "pre-commit-hook": "yarn lint-staged",
     "format": "prettier --write",
     "build": "yarn workspace @web3-ui/components build",
-    "test": "yarn workspace @web3-ui/components test"
+    "test": "jest --maxWorkers=2",
+    "test:watch": "yarn test --watch",
+    "test:coverage": "jest --coverage --colors --maxWorkers=2"
```

Comment:
We should probably adjust our use of `--maxWorkers` flags here. `2` is oddly specific and imo doesn't maximize things in our favor: https://dev.to/vantanev/make-your-jest-tests-up-to-20-faster-by-changing-a-single-setting-i36